.travis.yml

```diff
@@ -22,7 +22,7 @@ matrix:
     # This environment tests that scikit-learn can be built against
     # versions of numpy, scipy with ATLAS that comes with Ubuntu Trusty 14.04
     - env: DISTRIB="ubuntu" PYTHON_VERSION="2.7" CYTHON_VERSION="0.23.4"
-           COVERAGE=true
+           COVERAGE=true TEST_DOCSTRINGS="false"
```

Comment:
No need to set TEST_DOCSTRINGS if you don't want to test the docstrings (similar to what we do with COVERAGE)